integration_test/third_party_apps_data/applications/postgresql/metadata.yaml

```diff
@@ -31,31 +31,145 @@ minimum_supported_agent_version:
 supported_operating_systems: linux
 supported_app_version: ["10.18+"]
 expected_metrics:
-  - type: workload.googleapis.com/postgresql.backends
+  - kind: GAUGE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.backends
+    value_type: INT64
+    representative: true
+  - kind: CUMULATIVE
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.allocated
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      source: backend|backend_fsync|checkpoints|bgwriter
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.writes
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      type: requested|scheduled
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.checkpoint.count
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      type: sync|write
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.duration
     value_type: INT64
-    kind: GAUGE
+  - kind: CUMULATIVE
     monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.maxwritten
```

Comment:
The scope [doc](https://docs.google.com/document/d/1M9r8CqZl2W9fYz8brzlWntEAqJUC1K43LHGvGkbpiPs/edit) has this as `workload.googleapis.com/postgresql.bgwriter.maxwriten.count`

Is this an intentional change to the scope doc? If so, can we send a suggested edit to the scope doc and have the approver double check it? If not, can we fix the translation so it matches the scope doc?

Reply:
Intentional change, due to this https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13328#discussion_r945933195

Will make a suggestion